packages/lit-dev-server/package.json

```diff
@@ -33,13 +34,30 @@
       ],
       "clean": "if-file-deleted"
     },
-    "start": {
-      "command": "node lib/server.js",
+    "start:local:main": {
+      "command": "LITDEV_ENV=local MODE=main node lib/server.js",
```

Comment:
It would be cool to simplify this somehow. In my 11ty one I had to duplicate all my rules to just add `ENV=prod` to all the commands